imagekitservice/client.go

```
@@ -0,0 +1,149 @@
+type UploadAuth struct {
+	Token         string        `json:"token"`
+	ExpiresAt     int64         `json:"expiresAt"`
+	UploadURL     string        `json:"uploadUrl"`
+	UploadPayload UploadPayload `json:"uploadPayload"`
+}
+
+var (
+	sdk         *imagekitsdk.Client
+	publicKey   string
+	privateKey  string
+	urlEndpoint string
```

golangci-lint (2.12.2)

[error] var urlEndpoint is unused

(unused)